software/test_project/src/game/Game.java

```diff
@@ -46,11 +48,26 @@ public Game(GraphicsManager gm){
 		avatar.setGame(this);
 		// add avatar to entities
 		entities.add(avatar);
-		
 		mapElements = new ArrayList<>();
-		
-//		theGreatWall = new ElementWall(100, 100, 50, 20, new Color(0, 255, 0));
-		theGreatWall = new ElementWall(15, 15, new Color(0,255,0));
+
+
+		// BEWARE=========================================================================================
+//====================================================================================================
+// 						DANGER
+
+		char[][] map=MapSaver.readMap("map_01");
+		for(int col=0; col<Constants.WINDOW_MAP_WIDTH/16; col+=1){
+			for(int row=0; row<Constants.WINDOW_MAP_HEIGHT/16; row+=1){
```

Comment:
Lines 54-68 should be added in a different method since this map-loading should not be always done when starting the game.
Also add some comments what this code should do!

Also, this code is not using the correct constants, i.e.
```suggestion
			for(int row=0; row<Constants.WINDOW_MAP_HEIGHT / Constants.MAP_ELEMENT_SIZE; row+=1){
```